mqtt/mqtt-edgehub/src/auth/authentication/edgehub.rs

```diff
@@ -69,7 +72,17 @@ impl Authenticator for EdgeHubAuthenticator {
         &self,
         context: AuthenticationContext,
     ) -> Result<Option<AuthId>, Self::Error> {
-        let auth_id = self.authenticate(context).await?;
+        let auth_id = (|| async {
+            info!("authenticate client");
+            let auth_id = self.authenticate(&context).await?;
+            Ok(auth_id)
+        })
+        .retry(ExponentialBackoff {
+            max_elapsed_time: Some(Duration::from_secs(60)),
+            ..ExponentialBackoff::default()
```

Comment:
dot dot syntax means ignore other fields?

Reply:
yeah, similar to spread operator in TS/JS or other languages. https://cheats.rs/#data-structures

Reply:
Here are the defaults used:
https://docs.rs/backoff/0.2.1/backoff/default/index.html

Here is a table showing it will retry 10 times in one minute with the default settings:
https://docs.rs/backoff/0.2.1/backoff/index.html